earth_moving/ral/sensor/sensor_backend.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from abc import ABC, abstractmethod
+
+
+class BaseSensorBackend(ABC): 
```

Comment:
No, not really; I just found it a good answer to the question: _what if I know there will be this method in the children, but it will be different in each of them?_ 

Also, how can you avoid using it? Just with a _pass_ or a _raise.NotImplementedError()_ ?

Reply:
Yep, `raise NotImplementedError` and `pass` indeed.
`ABC` is probably the better recommended way so let's go with it, I'll read a little more about it.